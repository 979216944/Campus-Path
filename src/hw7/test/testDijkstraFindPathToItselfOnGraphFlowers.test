# Test Dijkstra finding a path,
# from a character in the dataset(flowers.tsv) to itself.

# Load graph
LoadGraph flowers flowers.tsv

# Find a minimum-cost path to itself
FindPath flowers Aconitum Aconitum
FindPath flowers Alchemilla Alchemilla
FindPath flowers Quince Quince
FindPath flowers Gaillardia Gaillardia