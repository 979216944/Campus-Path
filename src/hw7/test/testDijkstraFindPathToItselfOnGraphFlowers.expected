# Test Dijkstra finding a path,
# from a character in the dataset(flowers.tsv) to itself.

# Load graph
loaded graph flowers

# Find a minimum-cost path to itself
path from Aconitum to Aconitum:
total cost: 0.000
path from Alchemilla to Alchemilla:
total cost: 0.000
path from Quince to Quince:
total cost: 0.000
path from Gaillardia to Gaillardia:
total cost: 0.000